pkg/kubelet/volume/host_path/host_path.go

```diff
@@ -54,6 +54,12 @@ func (plugin *hostPathPlugin) CanSupport(spec *api.Volume) bool {
 	return false
 }
 
+func (plugin *hostPathPlugin) GetAccessModes() []api.AccessModeType {
+	return []api.AccessModeType{
+		api.ReadWriteOnce,
```

Comment:
is gitRepo read/write?  I got the impression it was read only after it cloned a repo.  

hostPath is not RW-Many because it is single host.  It is ReadWriteOnce.  Something like NFS can be ReadWriteMany.